lib/metered_raven_error_handler.rb

```diff
@@ -3,8 +3,14 @@
 
 module SidekiqMeteredExceptions
   class MeteredRavenErrorHandler < ::Raven::SidekiqErrorHandler
+    attr_reader :errors_to_ignore_on_first_occurrence
+
+    def initialize(errors_to_ignore_on_first_occurrence: [])
+      @errors_to_ignore_on_first_occurrence = errors_to_ignore_on_first_occurrence
+    end
+
     def call(ex, original_context)
-      ::Rails.logger.debug("MeteredRavenErrorHandler -- Error on Sidekiq job. Exception: #{ex.inspect} - Context: #{original_context.inspect}")
+      logger.debug("MeteredRavenErrorHandler -- Error on Sidekiq job. Exception: #{ex.inspect} - Context: #{original_context.inspect}")
```

Comment:
It's unclear if Sidekiq provides access to `logger` object on error handler ([it does so for workers](https://github.com/mperham/sidekiq/wiki/Logging)). Will test before merging, but would prefer to keep the gem agnostic to whether it's used from rails or any other app